bin/build.sh

```diff
@@ -0,0 +1,65 @@
+#!/usr/bin/env bash
+
+# Prevent jenkins from immediately killing the script when a step fails, allowing us to notify github:
```

Comment:
It's building:
- CLI + Spark universe (uploaded to S3)
- Docker image (pushed to dockerhub)

Added explanation in the file header comments